packages/core/src/env.ts

```diff
@@ -143,7 +143,7 @@ export const env: Env = cleanEnv(
 		SETTING_INVITES_ENABLED: bool({ default: false }),
 		SETTINGS_REGISTRATIONS_REQUIRED_ON_START: bool({ default: false }),
 		ADMIN_PASSWORD_RESET: bool({ default: false }),
-		FAKE_DATA_GENERATOR: bool({ default: false }),
+		FAKE_DATA_GENERATOR: bool({ default: true }),
```

Comment:
@sunk0 please revert such change. ALWAYS check each file you are changing before you push code